api/XDS_PROTOCOL.md

```diff
@@ -291,10 +291,15 @@ static_resources:
   - name: ads_cluster
     connect_timeout: { seconds: 5 }
     type: STATIC
-    hosts:
-    - socket_address:
-        address: <ADS management server IP address>
-        port_value: <ADS management server port>
+    load_assignment:
+      cluster_name: ads_cluster
```

Comment:
I wonder if we should make `cluster_name` optional for these embedded scenarios? This information is strictly redundant. @dio @mattklein123 ?

Reply:
@htuch do you think [this](https://github.com/envoyproxy/protoc-gen-validate/issues/209) does make sense?

Reply:
@dio sort of. I think we can do this today by removing the check from PGV and putting it in Envoy at the consuming sites. I'm not sure how to do it generically though, ideas?